api/v01/vrp.rb

```diff
@@ -75,10 +75,32 @@ class Vrp < APIBase
           desc 'Submit VRP problem', {
 
             nickname: 'submit_vrp',
-            success: VrpResult,
-            failure: [
-              { code: 400, message: 'Bad Request', model: ::Api::V01::Status }
-            ],
+            success: [{
+              code: 200,
+              message: 'The VRP has been processed synchronously',
+              model: VrpSyncJob
+            }, {
+              code: 201,
+              message: 'The VRP has been placed in a queue to be processed',
+              model: VrpAsyncJob
+            }],
+            failure: [{
+                code: 400,
+                message: 'Bad Request',
+                model: ::Api::V01::Status
+              }, {
+                code: 401,
+                message: 'Unauthorized',
+                model: ::Api::V01::Status
+              }, {
+                code: 402,
+                message: 'Subscription expired',
+                model: ::Api::V01::Status
+              }, {
+                code: 413,
+                message: 'Exceeded limit authorized for your account. Please contact support or sales to increase limits.',
```

Comment:
We do not need to do exactly the same but for user quotas google uses [402](https://developers.google.com/webmaster-tools/search-console-api-original/v3/errors#PAYMENT_REQUIRED) and [413](https://developers.google.com/webmaster-tools/search-console-api-original/v3/errors#REQUEST_ENTITY_TOO_LARGE) is literally related to size of the data.

Reply:
But again they differentiate between the http status code and a separate error code. We don't have an error code but we give the error message directly..

Reply:
The current PR has only for purpose to document all the http error codes. We currently have no such shade on errors except for http code 500 such as ClusteringError, DiscordantProblemError, JobKilledError, SchedulingHeuristicError or UnsupportedRouterModeError. But error code 500 probably requires its own PR.

Moreover grape-swagger and swagger-ui doesn't look like to fulfill this need.